src/config.py

```diff
@@ -0,0 +1,27 @@
+import os, dotenv
+from .logger import Logger
+
+class Config:
+    def __init__(self,
+                 logger=None):
+        # Load the dotenv configuration & set defaults
+        dotenv.load_dotenv()
+        self.set_defaults()
+
+        # Parse arguments & set logger
+        self.logger = logger if logger else Logger(
+            logger_name="default", logfile=self.logfile).logger
+
+    def set_defaults(self):
+        """
+        Set default global required parameters if none was found
+        """
+        # set logs file
+        self.logfile = os.getenv("OLS_LOGFILE", "logs/ols.logs")
```

Comment:
i guess if i don't want file logging i need to set OLS_LOGFILE=/dev/null?

because if i don't set it at all, this default will take effect on me.  And i can't set it to "empty string" (or at least i'm not sure what would happen if i did....)

Reply:
Actually, if OLS_LOGFILE is None the logger will not log to file, but this one is forcing it to its current behavior. Ideally this should just disable it by default unless explicitly enabled (because it is targeting a container)

Reply:
> Actually, if OLS_LOGFILE is None the logger will not log to file

yeah, but as a user who's just invoking the python app, i cannot set it to "None", because all i can do is set the env variable, or not set the env variable.  That's my point.

Reply:
(and if i don't set the env variable, then this code is going to default a value)

Reply:
i think you've updated this to default to None now, so people who don't want file logging can just not set the env var.  resolving.